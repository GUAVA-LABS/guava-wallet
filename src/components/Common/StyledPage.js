import styled from 'styled-components';

const StyledPage = styled.div`
    .ant-card {
        background: #ffffff;
        box-shadow: 0px 1px 3px 0px rgba(0, 0, 0, 0.04);
        overflow: hidden;

        &:hover {
            box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.1);
        }

        * {
            color: rgb(62, 63, 66);
        }

        .ant-card-head {
            color: #6e6e6e !important;
            background: #fbfcfd;
            border-bottom: 1px solid #eaedf3;
        }

        .ant-alert {
            background: #fbfcfd;
            border: 1px solid #eaedf3;
        }
    }
    .ant-card-body {
        border: none;
    }
    .ant-collapse {
        background: #fbfcfd;
        border: 1px solid #eaedf3;

        .ant-collapse-content {
            border: 1px solid #eaedf3;
            border-top: none;
        }

        .ant-collapse-item {
            border-bottom: 1px solid #eaedf3;
        }

        * {
            color: rgb(62, 63, 66) !important;
        }
    }
`;

export default StyledPage;
