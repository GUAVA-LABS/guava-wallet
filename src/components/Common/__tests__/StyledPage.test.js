import React from 'react';
import renderer from 'react-test-renderer';
import StyledPage from '../StyledPage';

test('Render StyledPage component', () => {
    const component = renderer.create(<StyledPage />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
