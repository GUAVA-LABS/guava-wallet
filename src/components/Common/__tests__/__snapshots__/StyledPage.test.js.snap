// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render StyledPage component 1`] = `
<div
  className="sc-bdVaJa dqgRVt"
/>
`;
